CharacterCreation/01_01_Races_Human

```diff
@@ -0,0 +1,145 @@
+==============================
+Humans
+==============================
+
+===== Society =====
+
+Humans originated on the planet Earth in the Sol system. They have been exploring space 
+to some degree for almost 800 years, settling their own Terran Empire after the catastrophic 
+events that changed the Earth.
+In general, adult humans range from 5’ to 6’5”. They can be larger or smaller, but individuals
+of those types are generally rare. 
+They are an incredibly prolific and opportunistic species, having completely terraformed and 
+colonized over half a dozen planets and their respective moons. With many more Terran planets 
+on the rise, they already dwarf the small Opalite and Tundari civilizations.
+
+Humans are very culturally dissimilar to each other. Even more so than most other species, 
+most likely due to their rapid expansion. As such, it is impossible to pick out the cultural 
+traits that influence humans as a whole. Central government is important to the large groups of 
+humans, though smaller factions and settlements on the rim of their empire do not seem to care 
+about the rules and regulations of their governing powers.
+
+
+Types:
```

Comment:
I may not have noticed before, but when I see "types" it makes me think... "something distinct that you only have 1 of." I.e., There is a biological type of human as well as a mammalian type of human, but they aren't the same thing. Not sure if that's just me, or wat.

Reply:
I've definitely always read it in the sense of qualities, as in humans have each of these listed things. Unfortunately it seems that the potential for confusion exists. Would "Qualities" be a better word? (Attributes is taken)

Reply:
It was super-clear to me. Maybe just 'cause it looks the same as the monster manuals from DnD, or the type from Pokemon. Types seems way more clear to me than Qualities.

Reply:
This is in place in order to designate how each species functions. It might seem redundant for humans because we're a little more familiar with humans than the alien species, but I have the same line in the Tundari Document.